src/Cell.tsx

```diff
@@ -141,7 +146,7 @@ function Cell<R>({
       rowIdx,
       row: rowData,
       isRowSelected,
-      onRowSelectionChange,
+      onRowSelectionChange: isSummaryRow ? noop : onRowSelectionChange,
```

Comment:
fwiw our checkbox column doesn't display any checkbox for summary rows, so maybe we don't need this check here.
And maybe users might want to display set up a custom column/row selection behavior.

Reply:
Makes sense, I have removed it